### PR TITLE
Fix profile warning in getting started doc

### PR DESCRIPTION
**Pull request by frankbu**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/setup/getting-started/index.md [hunk 1]
```diff
@@ -79,6 +79,12 @@ Follow these steps to get started with Istio:
     selected to have a good set of defaults for testing, but there are other
     profiles for production or performance testing.
 
+    {{< warning >}}
```
Comment (sdake) [inline content/en/docs/setup/getting-started/index.md:82]: An idea block may be preferable.

**Review by sdake — APPROVED** [review]
Nice work @frankbu . Appreciate that you took the time to improve this in a way that is mergeable in the documentation.

**Comment by sdake** [comment]
@hickeyma not sure how this impacts your work. PTAL.

File: content/en/docs/setup/getting-started/index.md [hunk 2]
```diff
@@ -79,6 +79,12 @@ Follow these steps to get started with Istio:
...
+    If your platform has a custom demonstration profile, e.g., Openshift, use
```
Comment (ericvn) [inline content/en/docs/setup/getting-started/index.md:83]: We call the profiles different things in different places: ex: Some additional vendor-specific configuration profiles are also available. For a complete list, refer to Vendor Optimized Profiles.

Comment (frankbu) [inline content/en/docs/setup/getting-started/index.md:83]:
```suggestion
    If your platform has a vendor-specific configuration profile, e.g., Openshift, use
```

File: content/en/docs/setup/getting-started/index.md (hunk 1 above)
Comment (frankbu) [reply content/en/docs/setup/getting-started/index.md:82]: IIUC, the demo profile doesn't work in OpenShift, so a warning is more appropriate.

**Review by ericvn — APPROVED** [review]

**Pull request merged**